### ISSUE TITLE
[Research] wPOKT Migration Design, Preparation & Research

### ISSUE THREAD

**Issue by Olshansk** [pokt-network/poktroll#461]
## Objective

Unblock work related to the wPOKT migration effort.

## Origin Document

- The [wPOKT migration discord channel](https://discord.com/channels/553741558869131266/1187104966826008696) with [RaidGuild](https://www.google.com/search?q=raidguild&rlz=1C5MACD_enUS1047US1047&oq=raidguild&gs_lcrp=EgZjaHJvbWUqBwgAEAAYgAQyBwgAEAAYgAQyDwgBEAAYChiDARixAxiABDIJCAIQABgKGIAEMgkIAxAAGAoYgAQyCQgEEAAYChiABDIJCAUQABgKGIAEMgkIBhAAGAoYgAQyCQgHEAAYChiABDIJCAgQABgKGIAEMgkICRAAGAoYgATSAQgxNDYzajBqN6gCALACAA&sourceid=chrome&ie=UTF-8).
- [Migration Proposal](https://hackmd.io/EQKZRTdPSyifly3CLC1oVA)

## Goals

- Research and understand the best (most secure, correct, extendable, least lock-in) mechanism to migrate the wPOKT bridge on Morse to Shannon
- Unblock RaidGuild from taking next steps on a design and implementation for the bridge migration

## Deliverables

- [ ] A 1-3 pager summarizing the suggested scope of work and changes to the proposal w/ reasoning and rationale

## Non-goals / Non-deliverables

- Any implementation related to the bridge
- A deep-dive into the entire industry on bridges
---

**Creator**: @Olshansk

**Comment by Olshansk** [comment]
[Reviewing_wPOKT_Token_Migration_6fc50c2e972842a381567da495492fbe.pdf](https://github.com/pokt-network/poktroll/files/14950986/Reviewing_wPOKT_Token_Migration_6fc50c2e972842a381567da495492fbe.pdf)

Closing the loop on this effort

![Screenshot 2024-04-11 at 1 19 21 PM](https://github.com/pokt-network/poktroll/assets/1892194/b46b28c0-43ba-4d3a-a25d-3f52453e2dc1)
